eventtracking/processors/whitelist.py

```diff
@@ -0,0 +1,21 @@
+"""Filter out events whose names aren't on a pre-configured whitelist"""
+
+from eventtracking.processors.exceptions import EventEmissionExit
+
+
+class WhitelistProcessor(object):
+    """
+
+    Filter out events whose names aren't on a pre-configured whitelist.
+
+    `whitelist` is an iterable containing event names that should be allowed to pass.
+    """
+
+    def __init__(self, **kwargs):
+        self.whitelist = frozenset(kwargs.get('whitelist', []))
```

Comment:
Wondering if we should make the whitelist required -- thinking that the tradeoff between providing an empty default (that lets nothing through) is outweighed by the possibility of misconfiguring.  I.e. if they want an empty whitelist, we require that they pass one.